src/Files.App/ViewModels/MainPageViewModel.cs

```diff
@@ -80,10 +75,11 @@ public MainPageViewModel()
 			OpenSettingsCommand = new RelayCommand(OpenSettings);
 		}
 
-		private void NavigateToNumberedTabKeyboardAccelerator(KeyboardAcceleratorInvokedEventArgs e)
+		private void NavigateToNumberedTabKeyboardAccelerator(KeyboardAcceleratorInvokedEventArgs? e)
 		{
 			int indexToSelect = 0;
-
+			if (e is null)
```

Comment:
Is e ever null?

Reply:
Just checked and it is never null.
Should I remove nullable annotations or should I use `!` to ignore them?

Reply:
👍